subsys/zigbee/cli/zigbee_cli_cmd_zcl.c

```diff
@@ -58,6 +58,33 @@
 
 #define ZCL_HELP "ZCL subsystem commands."
 
+#define GROUPS_ADD_HELP \
+	("Add the endpoint on the remote node to the group.\n" \
```

Comment:
You may consider aligning this message with the CLI command reference (the other commands are aligned).

Reply:
Found that, done that